doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
The issue with the one class is that every time a new coordinate system is initialized is the we would have to define the differential_class which could become cumbersome.  If we do implement subclasses the Lambda() part could be avoided for known systems.  If a general CurvilinearSys() subclass is implemented (in a future PR) the user would need to define certain parameters for the coordinate system they want.